source/infrastructure/add-new-aws-users.html.md.erb

```diff
@@ -1,15 +1,15 @@
 ---
 title: Add new AWS users
 weight: 10
-last_reviewed_on: "2022-07-13"
+last_reviewed_on: "2022-11-28"
 review_in: 6 months
 ---
 
 # Add new AWS users
 
 You can add new AWS users to the GovWifi AWS accounts by:
 
-1. First verify with the GovWifi developers whether to grant `admin` or `read-only` access.
+1. First verify with the GovWifi developers whether to grant `admin` or `read-only` access. Only team members with SC clearance can be allowed `admin` access. Until SC clearance is passed `read-only` access is allowed.
```

Comment:
A tiny nit pick - I _think_ the second clearance is redundant.

Reply:
Good spot! I'll make a quick commit to fix that.